kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiBatchSession.scala

```diff
@@ -207,7 +207,6 @@ class KyuubiBatchSession(
 
   override def close(): Unit = {
     super.close()
```

Comment:
`super.close()` only closes running operations, so it does not include `batchJobSubmissionOp` before `batchJobSubmissionOp` runs.